src/Compilers/CSharp/Test/Syntax/Parsing/RawStringLiteralParsingTests.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    public class RawStringLiteralParsingTests : ParsingTests
+    {
+        public RawStringLiteralParsingTests(ITestOutputHelper output) : base(output)
```

Comment:
nit: was this useful?
We typically use this for parsing tests (see `FieldNamedData` as illustration) as the machinery for `UsingTree`/`UsingExpression` outputs the correct syntax tree into the console output

Reply:
Gentle ping

Reply:
changing this.